### DISCUSSION TITLE
Using pip to install/update kobodl

### DISCUSSION THREAD

**Discussion by puffenuff** [subdavis/kobo-book-downloader#91] [Q&A]
When i use the command 

pip install kobodl

it automatically installs kobodl 0.8.0

How can I direct it to install 0.9.2? Thanks in advance. 
This is after unistalling and attempting to reinstall kobodl hence the requirements already satisfied for the other line items. 

pip install kobodl
Collecting kobodl
  Using cached kobodl-0.8.0-py3-none-any.whl (26 kB)
Requirement already satisfied: dataclasses-json<0.6.0 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from kobodl) (0.5.7)
Requirement already satisfied: pyperclip in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from kobodl) (1.8.2)
Requirement already satisfied: tabulate<0.9.0 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from kobodl) (0.8.10)
Requirement already satisfied: dataclasses<1.0.0 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from kobodl) (0.6)
Requirement already satisfied: beautifulsoup4<5.0.0 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from kobodl) (4.11.1)
Requirement already satisfied: flask>=1.1.0 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from kobodl) (2.0.3)
Requirement already satisfied: pycryptodome<4 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from kobodl) (3.15.0)
Requirement already satisfied: click<8 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from kobodl) (7.1.2)
Requirement already satisfied: requests>=2.25 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from kobodl) (2.28.1)
Requirement already satisfied: soupsieve>1.2 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from beautifulsoup4<5.0.0->kobodl) (2.3.2.post1)
Requirement already satisfied: marshmallow-enum<2.0.0,>=1.5.1 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from dataclasses-json<0.6.0->kobodl) (1.5.1)
Requirement already satisfied: typing-inspect>=0.4.0 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from dataclasses-json<0.6.0->kobodl) (0.8.0)
Requirement already satisfied: marshmallow<4.0.0,>=3.3.0 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from dataclasses-json<0.6.0->kobodl) (3.18.0)
Requirement already satisfied: Werkzeug>=2.0 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from flask>=1.1.0->kobodl) (2.2.2)
Requirement already satisfied: Jinja2>=3.0 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from flask>=1.1.0->kobodl) (3.1.2)
Requirement already satisfied: itsdangerous>=2.0 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from flask>=1.1.0->kobodl) (2.1.2)
Requirement already satisfied: idna<4,>=2.5 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from requests>=2.25->kobodl) (3.4)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from requests>=2.25->kobodl) (1.26.12)
Requirement already satisfied: certifi>=2017.4.17 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from requests>=2.25->kobodl) (2022.9.24)
Requirement already satisfied: charset-normalizer<3,>=2 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from requests>=2.25->kobodl) (2.1.1)
Requirement already satisfied: MarkupSafe>=2.0 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from Jinja2>=3.0->flask>=1.1.0->kobodl) (2.1.1)
Requirement already satisfied: packaging>=17.0 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from marshmallow<4.0.0,>=3.3.0->dataclasses-json<0.6.0->kobodl) (21.3)
Requirement already satisfied: mypy-extensions>=0.3.0 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from typing-inspect>=0.4.0->dataclasses-json<0.6.0->kobodl) (0.4.3)
Requirement already satisfied: typing-extensions>=3.7.4 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from typing-inspect>=0.4.0->dataclasses-json<0.6.0->kobodl) (4.4.0)
Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages (from packaging>=17.0->marshmallow<4.0.0,>=3.3.0->dataclasses-json<0.6.0->kobodl) (3.0.9)
Installing collected packages: kobodl
Successfully installed kobodl-0.8.0

**Comment by kkuyper** [comment]
Same issue here. Only possible to retrieve the 0.8.0 version?

**Reply by kkuyper** [reply]
I found the issue: I had the latest version of Python (3.10.x), but for kobodl you need an earlier version. That installed the 0.9.2 version of kobodl on my machine.

![image](https://user-images.githubusercontent.com/17176374/197527548-1744ec58-5199-47d4-a614-15f4e4a406bd.png)
https://pypi.org/project/kobodl/

**Reply by puffenuff** [reply]
Thank you for this! Much appreciated. Got it work now.

**Reply by subdavis** [reply]
Supports 3.10 as of version 0.9.3
